src/js/elements/location_info_box.js

```diff
@@ -1,5 +1,6 @@
-import {Observable, numFmt} from '../utils';
+import {Observable, numFmt, formatNumericalValue} from '../utils';
 import {Profile} from '../profile';
+import {format as d3format} from "d3-format/src/defaultLocale";
```

Comment:
this is probably a leftover after the refactoring.

Reply:
removed 👍